src/state/info/queries/tokens/tokenData.ts

```diff
@@ -81,11 +81,11 @@ const fetchTokenData = async (
     return { data, error: false }
   } catch (error) {
     console.error('Failed to fetch token data', error)
-    return { erro: true }
+    return { error: true }
```

Comment:
This could have been an issue since error attribute in the object would be undefined if we return this object